packages/bsky/src/services/indexing/processor.ts

```diff
@@ -186,7 +187,7 @@ export class RecordProcessor<T, S> {
       if (!found) {
         return this.params.eventsForDelete(deleted, null)
       }
-      const record = JSON.parse(found.json)
+      const record = jsonStringToLex(found.json)
```

Comment:
think you need to do lex -> json on intake as well

Reply:
Oh hah! I was like "how is this not causing errors in the tests?!" It's cause you encouraged me to add the `toJSON` method to `BlobRef` 🙇 

We should still send it through the proper lex serialization step so that it can catch bytes/cids/any future updates. But pretty handy that that saved us 👌

Reply:
Oh dang, good catch—not sure how I missed those!